### PR TITLE
Added Rachael Ainsworth info to relevant pages

### PR DESCRIPTION
**Pull request by rainsworth**
### Summary

I added info about Rachael Ainsworth (myself) to relevant pages within the Turing Way repo (social media, contact, bio, etc).

### List of changes proposed in this PR (pull-request)

* Updated ways_of_working.md & contributors.md
* Added name/info to project_management/dissemination_plan.md & turing-webpage.md

### What should a reviewer concentrate their feedback on?

- [x] Is it ok for Rachael Ainsworth to be added to pages within the project_management folder?
- [x] Are the links right?
- [x] Did I use the right emojis in contributors.md for this? 😂

### Acknowledging contributors

By submitting this pull request I confirm that:

- [x] all contributors to this pull request who wish to be included are named in [contributors.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/contributors.md).

### CODE REVIEW

File: ways_of_working.md [hunk 1]
```diff
@@ -14,12 +14,13 @@ This documents outlines expectations and responsibilities of project members and
...
 | Alex Morley | Mozilla Fellow | not paid by project |  Nov 2018 - June 2019  |
 | Martin O'Reilly | Principal Research Software Engineer | 10 % | Nov 2018 - Mar 2019 |
+| Rachael Ainsworth | Research Associate | 100% | Apr 2019 - May 2019 |
 
 ## Communication
 We're always around for a chat in our public gitter channel: https://gitter.im/WhitakerLab/the-turing-way
 
 In addition, you can reach the team members through their preferred way of communication:
-- Kirstie Whitaker: Mention me on a Github issue or pull request ([@KirstieJane](https://github.com/martintoreilly)), or tag me in [Gitter](https://gitter.im/alan-turing-institute/the-turing-way), @KirstieJane. Don't be afraid to nudge if I've not replied after a day or two! :sparkling_heart:
+- Kirstie Whitaker: Mention me on a Github issue or pull request ([@KirstieJane](https://github.com/KirstieJane)), or tag me in [Gitter](https://gitter.im/alan-turing-institute/the-turing-way), @KirstieJane. Don't be afraid to nudge if I've not replied after a day or two! :sparkling_heart:
```
Comment (KirstieJane) [inline ways_of_working.md]: Nice catch!

**Comment by KirstieJane** [comment]
@rainsworth - merge when you're ready!

**Comment by rainsworth** [comment]
@KirstieJane I don't have write access to this repo so unable to merge myself 🙈

**Comment by pherterich** [comment]
Will merge for you and opened #355 to get you sorted with permissions @rainsworth

**Pull request merged**